### PR TITLE
Add questions of 3 terms before the fall 2018 semester

### PR DESCRIPTION
**Pull request by mahlashrifi**

### CODE REVIEW

**Comment by smf8** [comment]
This PR partly resolves #6 which is already closed. After merging a new page will be created for topic-based categorization. @1995parham  We can merge it if you are ok with it.

File: Fall-2015/HW-5/p1/p1.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ب) تابعی بنويسيد كه عدد x را بگيرد، اگر x يك جمله در اين سری باشد شماره جمله آن به عنوان نتيجه توليد شود و در غير اين صورت عدد 1- برگردانده شود. مثلا اگر x برابر 10 باشد، نتيجه برابر 4 خواهد بود.  
+ج) برنامه‌ای بنويسيد كه دو عدد a و b را از كاربر بگيرد و همه اعداد در بازه (a, b] را كه عضوی از اين سری هستند را با شماره جمله مشخص كند.  
+ تذکر:  
+ * بدیهی است، آنچه شما برای این سوال آپلود می‌کنید، پاسخ قسمت (ج) می‌باشد.
+
+
+ مثال:  
+
+input:
+
```
Comment (1995parham) [inline Fall-2015/HW-5/p1/p1.md:16]: These should be in a code block with the backtick.

File: Fall-2015/HW-5/p3/p3.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+مساله برج هانوی و روش حل بازگشتي آن را از [ويكی پديا](https://fa.wikipedia.org/wiki/%D8%A8%D8%B1%D8%AC_%D9%87%D8%A7%D9%86%D9%88%DB%8C) مطالعه كنيد. هدف از اين تمرين اين است كه مراحل حل اين مساله برای n حلقه (كه تعداد آن از كاربر گرفته ميشود) را به صورت شكل‌های زير نمايش دهيد.  
+به طور **کلی** (توجه کنید که این کد صرفا برای توضیح منطق الگوریتم آورده شده است) کد شما دارای ساختار زیر خواهد بود:  
+
+    void hanoi ( int nDisk, char start, char temp, char finish )
```
Comment (1995parham) [inline Fall-2015/HW-5/p3/p3.md]: Using backtick is better than space for code blocks and also you can specify the language.

File: Fall-2016/HW-1/p4/p4.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+فرض كنيد دستور `(random(x` عدد صحيح x را عنوان ورودی می‌گيرد و يک عدد صحيح را به صورت تصادفی در بازه [-x,x] توليد می‌كند. الگوريتمی بنويسيد كه دو عدد صحيح a و b را بگيرد و تمام اعداد در بازه [a,b] را به صورت تصادفی توليد كند.
+
+ورودی:
+
+     -4 5
+
+خروجی:
+
+```markdown
```
Comment (1995parham) [inline Fall-2016/HW-1/p4/p4.md]: This should be `sh`.

**Review by 1995parham — CHANGES_REQUESTED** [review]
Before merging this please apply the following changes:

- Always use backtick for codeblocks and specify the language.
- Inputs and outputs are codeblocks with `sh` language.

File: Fall-2015/HW-5/p1/p1.md (hunk 1 above)
Comment (mahlashrifi) [reply Fall-2015/HW-5/p1/p1.md:16]: Thanks for pointing this out . 
Has been Applied to all cases .

**Review by 1995parham — APPROVED** [review]

**Pull request merged**